packages/lesswrong/components/tagging/TagProgressBar.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { registerComponent, Components } from '../../lib/vulcan-lib';
+import { Posts } from '../../lib/collections/posts';
+import { useMulti } from '../../lib/crud/withMulti';
+import withErrorBoundary from '../common/withErrorBoundary';
+import LinearProgress from '@material-ui/core/LinearProgress';
+import { Link } from '../../lib/reactRouterWrapper';
+
+const styles = theme => ({
+  root: {
+    background: "white",
+    padding: 10,
+    paddingLeft: 12,
+    paddingRight: 12,
+    fontSize: "1.3rem",
+    boxShadow: theme.boxShadow,
+    ...theme.typography.postStyle
+  },
+  inner: {
+    width: "100%",
+  },
+  tooltip: {
+    display: "block"
+  },
+  link: {
+    color: theme.palette.primary.main
+  },
+  text: {
+    display: "flex",
+    justifyContent: "space-between",
+    marginBottom: 4,
+  }
+});
+
+const TagProgressBar = ({classes}: {
+  classes: ClassesType,
+}) => {
+
+  const { LWTooltip, PostsItem2MetaInfo } = Components;
+
+  const { totalCount: taggedTotal } = useMulti({
```

Comment:
This downloads 10 random posts that aren't displayed (and the other `useMulti` downloads another 10). Give this a `limit: 0` to fix that.